Dockerfile

```diff
@@ -255,7 +255,8 @@ RUN set -x \
 	&& cd "$GOPATH/src/github.com/opencontainers/runc" \
 	&& git checkout -q "$RUNC_COMMIT" \
 	&& make static BUILDTAGS="seccomp apparmor selinux" \
-	&& cp runc /usr/local/bin/docker-runc
+	&& cp runc /usr/local/bin/docker-runc \
+	&& rm -rf "$GOPATH"
```

Comment:
make sense, `/tmp` is not mount as `tmpfs`  during the build, we have to move the tmp file manually